app/services/register_user.rb

```diff
@@ -0,0 +1,11 @@
+class RegisterUser
+  attr_reader :github_id
+
+  def initialize(params)
+    @github_id = params[:github_id]
+  end
+
+  def perform
+    @_user = User.create(github_id: github_id)
```

Comment:
k cool, so let's memoize this (change to `@_user ||= User.create(github_id: github_id)`)